### PR TITLE
Add `Attaching Request Body` to Expected Features.

### PR DESCRIPTION
**Pull request by maciejwalkowiak**

### CODE REVIEW

**Comment by maciejwalkowiak** [comment]
cc @bruno-garcia

File: src/docs/sdk/features.mdx [hunk 1]
```diff
@@ -179,3 +179,15 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - HTTP request must be enhanced with a [`sentry-trace` HTTP header](/sdk/performance/#header-sentry-trace)
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
+
+## Attaching Request Body
```
Comment (bruno-garcia) [inline src/docs/sdk/features.mdx:183]:
```suggestion
## Attaching Request Body in Server SDKs
```

File: src/docs/sdk/features.mdx [hunk 2]
```diff
@@ -179,3 +179,15 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
...
+
+Ability for the SDK to attach request body to events and/or transactions triggered during the execution of request.
```
Comment (bruno-garcia) [inline src/docs/sdk/features.mdx:185]:
```suggestion
Ability for the SDK to attach request body to events and triggered during the execution of request.
```

File: src/docs/sdk/features.mdx [hunk 3]
```diff
@@ -179,3 +179,15 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
...
+
+User should be able to set a configuration option `maxRequestBodySize` to instruct SDK how big requests bodies should be attached.
+SDK controls what is an actual size in bytes for each option:
+
+- `none` (default)
+- `small` - `1000` bytes
+- `medium` - `10000` bytes
+- `always`
```
Comment (bruno-garcia) [inline src/docs/sdk/features.mdx:193]:
```suggestion
- `always`

Some examples: 
 * .NET: [Options](https://github.com/getsentry/sentry-dotnet/blob/46bcd9ffc79c7d750144a395c71b2cfc7a9f0e30/src/Sentry.AspNetCore/SentryAspNetCoreOptions.cs#L23-L30), [Implementation](https://github.com/getsentry/sentry-dotnet/blob/46bcd9ffc79c7d750144a395c71b2cfc7a9f0e30/src/Sentry/Extensibility/RequestBodyExtractionDispatcher.cs#L45-L71)
 * Java
 * Python
```

Comment (untitaker) [reply src/docs/sdk/features.mdx:193]: and nodejs

**Review by untitaker — APPROVED** [review]

File: src/docs/sdk/features.mdx (hunk 2 above)
Comment (maciejwalkowiak) [reply src/docs/sdk/features.mdx]: While technically it is possible to set it on transactions, no SDK does it at this stage.

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**